uui-editor/src/plate/plugins/tablePlugin/tablePlugin.tsx

```diff
@@ -173,7 +177,7 @@ const Table = (props: any) => {
 
     const renderToolbar = useCallback(() => {
         return (
-            <div className={ cx(css.imageToolbar, 'slate-prevent-blur') }>
+            <div className={ cx(imageBlockCss.imageToolbar, 'slate-prevent-blur') }>
```

Comment:
can we move this styles and may be wrapper also inside Toolbar component to not add them directly for tables and images?